nucleus-api/src/main/java/io/github/nucleuspowered/nucleus/api/nucleusdata/Kit.java

```diff
@@ -135,6 +135,26 @@ default Kit addCommand(String command) {
     Kit updateKitInventory(Player player);
 
     /**
+     * Redeems any items and commands in this kit for the specified player.
+     *
+     * @param player The player.
+     * @return <code>true</code> if the redemption was successful
+     */
+    default boolean redeem(Player player) {
+        boolean success = redeemKitItems(player);
+        if (success) redeemKitCommands(player);
+        return success;
```

Comment:
I'm not sure this would be better
```java
default boolean redeem(Player player) {
     if (redeemKitItems(player)) {
        redeemKitCommands(player);
        return true;
    } else {
        return false;
    }
}
```